questionnaires/views.py

```diff
@@ -183,6 +184,23 @@ def get_filename(self):
         return f'{self.user.username}-submission_{timestamp}'
 
 
+class ExperimentalFormDataPerUserView(FormDataPerUserView):
+    list_export = ['Page ID', 'Page Title', 'Submission ID', 'Submission Date', 'Field', 'Value']
+
+    def get_rows(self, item, form_fields_dict):
+        data = {
+            'User': item.user.username,
+            'URL': item.page.full_url,
+        }
+        for clean_name, answer in json.loads(item.form_data).items():
+            data.update({
+                form_fields_dict.get(item.page_id, {}).get(clean_name, clean_name): answer,
+            })
+
+        return [dict(zip(self.list_export, [item.page_id, item.page.title, item.id, item.submit_time, field, value]))
+                for field, value in data.items()]
```

Comment:
It seems to me that that only difference is the list of column headers (`list_export`) and their corresponding values.

I think the difference would be clearer if we extract the part that gets the column values:
```python
def get_values(self, item, field, value):
    return [item.page_id, item.page.title, item.id, item.submit_time, field, value]
```

And use it in `get_rows` in the appropriate place:
```python
def get_rows(self, item, form_fields_dict):
    # ...
    return [dict(zip(self.list_export, self.get_values(item, field, value)]))
                for field, value in data.items()]
```

Then we can just override `list_export` and `get_values()` for the new format.